libsmtutil/SolverInterface.h

```diff
@@ -42,21 +42,24 @@ namespace solidity::smtutil
 struct SMTSolverChoice
 {
 	bool cvc4 = false;
+	bool eld = false;
 	bool smtlib2 = false;
 	bool z3 = false;
```

Comment:
Not sure about the other tests yet but I think that [`t_archlinux_soltest
`](https://app.circleci.com/pipelines/github/ethereum/solidity/25900/workflows/85ffec54-b5d6-4a5f-a11b-904281e61ae4/jobs/1143820) is failing because the new option is not included in the expectations:

https://github.com/ethereum/solidity/blob/a229d7219bcc7100364ed9c636fc3bbabf4fb4bb/test/solc/CommandLineParser.cpp#L210-L219

But I wonder why the compiler is not complaining that the struct initializer is missing one item.